### PR TITLE
Add migrate-manifest command

### PR DESCRIPTION
**Pull request by appleton**
Add a command to read the `app-ci.json` and assign its contents to the `environments.test` key of an `app.json` file.

### CODE REVIEW

File: commands/ci/migrate-manifest.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  try {
+    appJSON = require(appJSONPath)
+  } catch (e) {
+    appJSON = {}
+  }
+
+  try {
+    appCiJSON = require(appCiJSONPath)
+  } catch (e) {
+    throw new Error(`Couldn't load app-ci.json. This command must be run from a directory containing an app-ci.json file`)
```
Comment (joshwlewis) [inline commands/ci/migrate-manifest.js]: This could probably be something more friendly. "This codebase isn't using app-ci.json, so you are good to go!"

Comment (joshwlewis) [inline commands/ci/migrate-manifest.js]: Basically, don't tell them they are doing something wrong, if they are using preferred configuration.

Comment (appleton) [inline commands/ci/migrate-manifest.js]: Good shout 👍

**Comment by raulb** [comment]
@joshwlewis @appleton Updated to cover all use cases explaining what has happened:

**If neither `app.json` nor `app-ci.json` exist:**

```
› heroku ci:migrate-manifest
We couldn't find an app-ci.json file in the current directory, but we're creating a new app.json manifest for you.
Creating app.json file... done
Please check the contents of your app.json before committing to your repo.
You should be all set now! 🎉
```

**If `app.json` exists with an `environments` key, and there isn't an `app-ci.json`:**

```
› heroku ci:migrate-manifest
We couldn't find an app-ci.json file in the current directory, and your app.json already has the environments key.
You should be all set now! 🎉
```

**If there is an `app-ci.json` and no `app.json`:**

```
› heroku ci:migrate-manifest
Creating app.json file... done
Deleting app-ci.json file... done
Please check the contents of your app.json before committing to your repo.
You should be all set now! 🎉
```

**If there is an `app-ci.json` and an `app.json` with no `environments` key:**

```
› heroku ci:migrate-manifest
Updating app.json file... done
Deleting app-ci.json file... done
Please check the contents of your app.json before committing to your repo.
You should be all set now! 🎉
```

**If there is an `app-ci.json` and a `app.json` with a `test` key:**

```
› heroku ci:migrate-manifest
 ▸    Your app.json already had a test key. We're overwriting it with the content of your app-ci.json
Updating app.json file... done
Deleting app-ci.json file... done
Please check the contents of your app.json before committing to your repo.
You should be all set now! 🎉
```

The `Please check the contents of your app.json before committing to your repo.` message is colored to highlight the action to take immediately after.

**Comment by appleton** [comment]
👍

**Pull request merged**